### DISCUSSION TITLE
Scrolling lists on Pixel 6 / Android

### DISCUSSION THREAD

**Discussion by ilx** [community/community#74560] [Other Feature Feedback, Questions, & Ideas]
It is really hard to scroll lists on Android if list fills the screen. Most of the time attempt to scroll moves handle from top of the screen instead of scroll, but sometimes it detects a scroll. I cannot figure out when list will scroll and when it will move window border down. Using it on Pixel 6 Android device.

**Comment by ilx** [comment]
Ok I found a pattern: if I am at the bottom of the list and attempt to scroll it will move that window handle down, but if I attempt to 1st scroll in the opposite direction it will not trigger a handle and I will be able to scroll.

I would suggest to replace this behaviour or the component as it is not great user experience..

**Reply by Jennifera982416** [reply]
Begging for
someone to help me for a long time now I neeed my info saved from 3 countries n really need my original email that got stolen  I've started over new phone new number new accounts whomever followed  I'm not a it person and I'm so lost in a nightmare I can wake up from

**Reply by Jennifera982416** [reply]
> Ok I found a pattern: if I am at the bottom of the list and attempt to scroll it will move that window handle down, but if I attempt to 1st scroll in the opposite direction it will not trigger a handle and I will be able to scroll.
> 
> I would suggest to replace this behaviour or the component as it is not great user experience..

Could you please help my Brand new
phone I haven't gotten to enjoy

**Reply by Jennifera982416** [reply]
I Cant even Screenshot I'm on a wrong domain as well as experimental software and I'm going nuts seriously 🙃 please someone o have 0 experience and don't know how to fix /

**Reply by Sameerali132** [reply]
Plz help me my mobile is hacked plz help

**Reply by Sameerali132** [reply]
> I Cant even Screenshot I'm on a wrong domain as well as experimental software and I'm going nuts seriously 🙃 please someone o have 0 experience and don't know how to fix /

Plz my all data secure plz help me my data is leak

**Reply by EMIL23GRA** [reply]
****

**Comment by TheMostOfIt** [comment]
- Block unwanted people from my phone
